util/helm.go

```diff
@@ -107,3 +107,12 @@ func CopyDir(src, dst string) error {
 	filepath.Walk(src, walker)
 	return failure
 }
+
+//CopyFile copies file from src to dst
+func CopyFile(src string, dst string) error {
+	data, err := ioutil.ReadFile(src)
```

Comment:
instead of reading the entire file into a buffer and then writing that buffer to another file, it's probably much more efficient to stream it using `io.Copy`:

```
func CopyFile(src, dst string) error {
    in, err := os.Open(src)
    if err != nil { return err }
    defer in.Close()
    out, err := os.Create(dst)
    if err != nil { return err }
    defer out.Close()
    _, err = io.Copy(out, in)
    cerr := out.Close()
    if err != nil { return err }
    return cerr
}
```